const.hpp

```diff
@@ -167,6 +167,18 @@ enum PelSeverity
     ERROR
 };
 
+enum SdRamType
+{
+    EN_DDR4 = 0x0C,
+    EN_DDR5 = 0x12
+};
+enum SdRamMixType
+
```

Comment:
Why this blank line here?

Reply:
> Why this blank line here?

clang-format was added, don't know why. I removed it